### PR TITLE
Update: schild-report.schild-report version 2.5.0.0

### PR DESCRIPTION
**Pull request by OfficialEsco**
- [X] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [X] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [X] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [X] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/26481)

### CODE REVIEW

**Review by ImJoakim — COMMENTED** [review]
Seems like the installer locale is German

image

**Review by jedieaston — APPROVED** [review]

File: manifests/s/schild-report/schild-report/2.5.0.0/schild-report.schild-report.locale.en-US.yaml [hunk 1]
```diff
@@ -7,7 +7,6 @@ PackageLocale: en-US
 Publisher: ${env.THE_AUTHOR}
```
Comment (jedieaston) [inline manifests/s/schild-report/schild-report/2.5.0.0/schild-report.schild-report.locale.en-US.yaml:7]: Is that really what the publisher's name is?

**Review by jedieaston — CHANGES_REQUESTED** [review]

File: manifests/s/schild-report/schild-report/2.5.0.0/schild-report.schild-report.locale.en-US.yaml (hunk 1 above)
Comment (ImJoakim) [reply manifests/s/schild-report/schild-report/2.5.0.0/schild-report.schild-report.locale.en-US.yaml:7]: Yup 🤦‍♂️

image

Comment (OfficialEsco) [reply manifests/s/schild-report/schild-report/2.5.0.0/schild-report.schild-report.locale.en-US.yaml:7]: Yes, Yes it is 😬
If you can help him he would be very happy 😁 https://github.com/schild-report/schild.report/issues/6

**Review by ImJoakim — APPROVED** [review]

**Pull request merged**